### DISCUSSION TITLE
Support for more type placeholder

### DISCUSSION THREAD

**Discussion by Darkseno** [Zrips/CMI#7771] [Feature Requests]
Hello !

With the last update some placeholders stopped working and it seems that I was very lucky because I was told that you don't take these placeholders into account.

I'm surprised, they worked for 1 year... anyway I'm tired of debating about this.

My issue will be clearer: https://github.com/Zrips/CMI/issues/7766

**Comment by CloudeLecaw** [comment]
i dont know how much people use this. 

But if it was working with CMI Chat before and stoped now working then of curse it would be a nice move
to add official support for it.

**Comment by Darkseno** [comment]
UP ?

**Reply by CloudeLecaw** [reply]
you dont need to up threads. if all people start to up there threads this would be just a waste.

Also uping does not make the proces of if it will be added or if it will be read faster.
